simcase-game/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "simcase-game",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.8.6",
```

Comment:
We need to previously declare all the requirements me need.
Don't forget there's a difference between _dependencies_ and _devDependencies_